distribution/docs/src/main/jdocs/content/_plugins/video-thumbnail-plugin-contents.adoc

```diff
@@ -22,4 +22,13 @@ The Video Thumbnail Plugin is installed by default with a standard installation
 
 ===== Configuring the Video Thumbnail Plugin
 
-The Video Thumbnail Plugin has no configurable properties.
+To configure the Video Thumbnail Plugin:
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-catalog}* application.
+. Select the *Configuration* tab.
+. Select the *Video Thumbnail Plugin*.
+
+The Video Thumbnail Plugin has one configurable property:
```

Comment:
Might be worth a note somewhere in the docs that increasing the size of video files that thumbnails are generated for can/will impact overall system performance.